manifests/init.pp

```diff
@@ -162,11 +162,22 @@
 
   include certs::pulp_client
 
+  file { '/etc/foreman':
```

Comment:
I still feel placing the certs in `$deployment_dir` or `${deployment_dir}/certs` is cleaner than `/etc/foreman`.